src/main/kotlin/com/launchdarkly/intellij/toolwindow/FlagNodeViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package com.launchdarkly.intellij.toolwindow
+
+import com.launchdarkly.api.model.FeatureFlag
+import com.launchdarkly.api.model.FeatureFlags
+import com.launchdarkly.intellij.LDIcons
+import com.launchdarkly.intellij.featurestore.FlagConfiguration
+
+class FlagNodeViewModel(
+    val flag: FeatureFlag,
+    val flags: FeatureFlags,
+    val flagConfig: FlagConfiguration?,
+) {
+    val description = flag.description
+    val isEnabled = flagConfig?.on == true
```

Comment:
see comment below, but to clarify that comment, if `flagConfig == null` then i think `isEnabled=null`

Reply:
oh okay, I see what you're saying, sorry I should have read this before responding to the other comment